mindmeld-ui/src/app/actions.js

```diff
@@ -0,0 +1,66 @@
+import * as APP from './constants';
+
+export const updateQuery = (query) => {
+  return {
+    type: APP.UPDATE_QUERY,
+    query,
+  };
+};
+
+export const executeQuery = () => {
+  return {
+    type: APP.EXECUTE_QUERY,
+  };
+};
+
+export const onExecuteQueryStart = () => {
+  return {
+    type: APP.ON_EXECUTE_QUERY_START,
+  };
+};
+
+export const onExecuteQueryEnd = (data) => {
+  return {
+    type: APP.ON_EXECUTE_QUERY_END,
+    data,
+  };
+};
+
+export const onExecuteQueryError = (error) => {
+  return {
+    type: APP.ON_EXECUTE_QUERY_ERROR,
+    error
+  };
+};
+
+export const resetState = () => {
+  return {
+    type: APP.RESET_STATE
+  };
+};
+
+export const startRecognition = (handler) => {
+    return {
+        type: APP.START_RECOGNITION,
+        handler,
```

Comment:
Is this a function? I think it's an anti pattern to include functions in action objects (except in support of thunks or async actions, or something else processed by a store middleware).

Reply:
Fixed